### ISSUE TITLE
How to get the path for the given module?

### ISSUE THREAD

**Issue by toranb** [ryanseddon/gulp-es6-module-transpiler#7]
First off -thank you for creating this awesome OSS project! It's already saved me a ton of time as I work to transition from grunt to gulp :)

In the grunt version of this module the "moduleName' is actually a function that provides the current path (usually the path on the file system in this case). How can I get something like this for the gulp version?

```js
    transpile: {
      app: {
        type: 'amd',
        moduleName: function(path) {
          return 'js/' + path;
        },
        files: [{
          expand: true,
          cwd: 'js/app/',
          src: '**/*.js',
          dest: 'js/dist/transpiled/app/'
        }]
      }
    }
```

**Comment by ryanseddon** [comment]
Is this fixed or still an issue in 0.2.0?

**Comment by toranb** [comment]
This solved my problem when you merged the PR listed

https://github.com/ryanseddon/gulp-es6-module-transpiler/pull/8
